EIPS/eip-cfi.md

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
```suggestion
---
eip: 7723
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

Reply:
> ```suggestion
> ---
> eip: 7723
> ```
> Assigning next sequential EIP/ERC/RIP number.
> 
> Please also update the filename.